lua/default-config.lua

```diff
@@ -488,6 +488,35 @@ lvim.lang = {
       },
     },
   },
+  fortran = {
+    formatters = {
+      {
+        -- @usage can be clang_format or uncrustify
+        exe = "",
+        args = {},
+        stdin = true,
```

Comment:
plz remove this line, we are using null-ls as formatting/linting provider
so just make it look like 
```lua
formatters = {
  -- {
  --  exe = "",
  --   args = {},
  -- },
},
```

Reply:
Since there are no formatters available, I would even say to put it like this
`formatters = {}`

Reply:
Fixed. I was copying from a different language.

Reply:
These are small changes done on the rolling branch, you might not be up-to-date with that one :)